_i18n/en.yml

```diff
@@ -326,13 +338,28 @@ help_pages:
 
       [Why do I need to use login.gov to access government services online?](site.baseurl/help/creating-an-account/why-do-i-need-to-use-logingov-to-access-government-services-online/)
     what-is-my-personal-key: |-
-      A personal key is a 16-character secret code we give you when you create an account. The personal key is the only way to sign in if you lose access to the device you use for two-factor authentication.
+      A personal key is a 16-character secret code we have been historically generating as a form of two factor authentication. [Read more about two-factor authentication here](site.baseurl/help/creating-an-account/what-is-two-factor-authentication/)
+
+      Recently we have began the process of retiring personal keys as a form of two factor authentication as they are apt to being lost easily. In their place users now have a variety of options for a second authentication factor including backup codes.
+
+      If you are creating a new account you will not be assigned a personal key. Instead, select one of the available two-factor authentication options presented.
 
-      You can only use your personal key once.  You will receive a new personal key each time you use it to sign in, and your old one will no longer work.
+      If you were assigned a personal key previously you are still able to use it, however next time you use the personal key you will be asked to select a new form of two factor authentication and the personal key option will be removed. Until you use your current personal key it will still be active and available to you as a two factor option.
```

Comment:
> Until you use your current personal key it will still be active and available to you as a two factor option.

Suggestion: Until you use your current personal key to sign in to your account, it will remain active and available as a second authentication factor.

Reply:
Changed